labelCloud/view/gui.py

```diff
@@ -638,3 +643,19 @@ def update_default_object_class_menu(self, new_classes: Set[str] = None) -> None
     def change_default_object_class(self, action: QAction) -> None:
         config.set("LABEL", "std_object_class", action.text())
         logging.info("Changed default object class to %s.", action.text())
+
+    def ask_custom_index(self):
+        input_d = QInputDialog(self)
+        self.input_pcd = input_d
+        input_d.setInputMode(QInputDialog.IntInput)
+        input_d.setWindowTitle("labelCloud")
+        input_d.setLabelText("Insert Point Cloud number: ()")
+        input_d.setIntMaximum(len(self.controller.pcd_manager.pcds)-1)
+        input_d.intValueChanged.connect(lambda val: self.update_dialog_pcd(val))
+        input_d.intValueSelected.connect(lambda val: self.controller.custom_pcd(val))
+        input_d.open()
+        self.update_dialog_pcd(0)
+    
+    def update_dialog_pcd(self, value) -> None:
+        pcd_path = self.controller.pcd_manager.pcds[value]
+        self.input_pcd.setLabelText("Insert Point Cloud number: ("+pcd_path.name+")")
```

Comment:
This looks better with an f-string:
`...Text(f"Insert Point Cloud number: {pcd_path.name}")`